File: src/main/java/botmanager/frostbalance/menu/settings/UserSettingsMenu.java
package botmanager.frostbalance.menu.settings;

import botmanager.frostbalance.Frostbalance;
import botmanager.frostbalance.command.MessageContext;
import botmanager.frostbalance.menu.Menu;
import net.dv8tion.jda.api.EmbedBuilder;
import org.jetbrains.annotations.NotNull;

public class UserSettingsMenu extends Menu {
    public UserSettingsMenu(Frostbalance bot, MessageContext context) {
        super(bot, context);
    }

    @NotNull
    @Override
    public EmbedBuilder getEmbedBuilder() {
        return new EmbedBuilder()
                .setTitle("User Settings");
    }
}

File: src/main/java/botmanager/doomstack/menu/settings/UserSettingsMenu.kt
package botmanager.doomstack.menu.settings

import botmanager.doomstack.DoomStack
import botmanager.doomstack.MessageContext
import botmanager.doomstack.menu.Menu
import botmanager.doomstack.menu.response.DynamicMenuResponse
import net.dv8tion.jda.api.EmbedBuilder

class UserSettingsMenu(bot: DoomStack, context: MessageContext) : Menu(bot, context) {

    init {

        //nothing lol

    }

    override val embedBuilder: EmbedBuilder
        get() {
            for (response in menuResponses) {
                if (response is DynamicMenuResponse) {
                    response.updateValues()
                }
            }
            return EmbedBuilder()
                    .setTitle("User Settings")
        }
}